llvm/lib/Transforms/Scalar/ConstraintElimination.cpp

```diff
@@ -273,7 +273,15 @@ class ConstraintInfo {
 
 public:
   ConstraintInfo(const DataLayout &DL, ArrayRef<Value *> FunctionArgs)
-      : UnsignedCS(FunctionArgs), SignedCS(FunctionArgs), DL(DL) {}
+      : UnsignedCS(FunctionArgs), SignedCS(FunctionArgs), DL(DL) {
+    auto &Value2Index = getValue2Index(false);
+    for (Value *V : FunctionArgs) {
```

Comment:
`// Add v > -1 constraints to unsigned system.` or something.

Reply:
Added, thanks!